tests/xfstests/run.pm

```diff
@@ -433,6 +436,13 @@ sub run {
     %BLACKLIST = (%BLACKLIST, %tests_needto_exclude);
 
     test_prepare;
+
+    # wait until nfs service is ready
+    if (get_var('PARALLEL_WITH')) {
+        mutex_wait('xfstests_nfs_server_ready');
+        script_retry('ping -c3 10.0.2.101', delay => 15, retry => 12);
```

Comment:
retry 12 feels generous; if mutex_wait works as it should, the retry should not be needed, right?
Anyway -c3 is there to have some lag time, so -c3 and retry 12 feels a bit steep imho

Reply:
Agree with 12 being too much. 
But IMHO `-c1` is enough, because wih `-c3` `ping` will fail if even just one of the three packets does not arrive, see:

[`man ping(8)`](https://linux.die.net/man/8/ping):
> If ping does not receive any reply packets at all it will exit with code 1. If a packet count and deadline are both specified, and fewer than count packets are received by the time the deadline has arrived, it will also exit with code 1.

Also, I double checked the code (https://github.com/iputils/iputils/blob/b3ba3ddcb4af52fb10d42d02cceb6fde9854e626/ping/ping_common.c#L935):
```
return (!rts->nreceived || (rts->deadline && rts->nreceived < rts->npackets));
```

`rts->nreceived < rts->npackets` is what we talk about.